src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/locations/availability/BookedLocationsService.kt

```diff
@@ -55,6 +55,8 @@ class BookedLocationsService(
 
     return when (activitiesAppointmentsClient.isAppointmentsRolledOutAt(prisonCode)) {
       true -> activitiesAppointmentsClient.getScheduledAppointments(prisonCode, date, nomisToDpsLocations.keys)
+        // Do not consider cancelled appointments
+        .filterNot { it.isCancelled }
```

Comment:
You could avoid the comment and use an extension function if you fancied, meh

`private fun List<AppointmentSearchResult>.doNotConsiderCancelledAppointments() = filterNot { it.isCancelled }`

or something to this effect.

Reply:
It's fairly self-explanatory anyway - I'll maybe take the comment out. Next pr..